content/departments/ce-support/support/process/how-to-tickets.md

```diff
@@ -0,0 +1,26 @@
+# Engaging Engineering
+
+As an Application Engineer, you will be confronted with tickets that require you to seek help from engineering. You are encouraged to do so! There are two ways to do so, largely depending on the nature of the ticket.
+
+### "How To" Tickets
+
+For "how-to" tickets, you are not required to create a Request for help (RFH) in GitHub. While RFH's are a valuable tool when dealing with complex issues, they introduce overhead and complexity that isn't necessary for "How-To" questions. Furthermore, they limit visibility, and collaboration between teams that happens in public, shared channels.
```

Comment:
Explaining how the old way of doing it is no longer necessary is helpful for people now, but it will be confusing going forward. I would move this below (and perhaps shorten it).